### ISSUE TITLE
Show chat notifications when clicking chat button

### ISSUE THREAD

**Issue by dorothy3242** [ppy/osu-web#5773]
having a button for chat but showing chat notifications in a different place is really weird, especially when the chat button shows the number of chat messages you have unread.

Clicking on the chat notifications under notifications also redirect you to the relevant chat, but clicking the chat button doesn't.

![2020-03-13_10-50-26](https://user-images.githubusercontent.com/33286824/76631816-aaf49e80-6518-11ea-8199-bc0559b5d8a1.gif)

so basically, proposed solution:

show _chat_ notifications when clicking the chat button, not the notifications button.
OR...
don't show number of chat messages on the chat button. Keep that for notifications only.

**Comment by peppy** [comment]
we already have a design for this and it will be added in time. the current button was added quickly for accessibility reasons.

![image](https://user-images.githubusercontent.com/191335/76673990-0a70ae00-65ee-11ea-8048-9adfc1ce2a13.jpeg)

**Comment by cl8n** [comment]
that image doesn't really answer the questions in OP. what's supposed to happen when you click the chat button? does the normal notification count include chat messages?

**Comment by peppy** [comment]
@arflyte

**Comment by arflyte** [comment]
Hmm, either we show a list of unread chats, or we remove the chat notification count at chat page.
